android/app/src/main/java/app/organicmaps/car/screens/PlaceScreen.java

```diff
@@ -279,7 +279,8 @@ public void showNavigation(boolean show)
   @Override
   public void onBuiltRoute()
   {
-    Framework.nativeDeactivatePopup();
+    // TODO: It is required to call this method to deactivate the popup. Unfortunately, due to the latest changes, calling this method cancels route building.
+    // Framework.nativeDeactivatePopup();
```

Comment:
@AndrewShkrob 

1. What will happen after merging this into the master? Pupups can't be closed anymore?
2. How should the right fix work?

CC @strump

Reply:
1. There will be a popup (selection) present after building the route: 
    <img width="200" alt="image" src="https://github.com/user-attachments/assets/147a2cb1-f4f3-46a1-91e9-4bf5e19afa2e">

2. A function named `deactivatePopup` should only deactivate the popup. It shouldn't close the place page, and it must not cancel route building.

Reply:
Thanks, let's call the blue circle on the map "selection" instead of confusing "popup".